client/helpers.go

```diff
@@ -236,9 +236,16 @@ func (c *Client) IsNotFoundError(err error) bool {
 	errorCode := ae.ErrorCode()
 	for _, s := range notFoundErrorPrefixes {
 		if strings.Contains(errorCode, s) {
-			c.logger.Warn("API returned \"NotFound\" error ignoring it...", "error", err)
```

Comment:
maybe keep the log line and use IsAWSErr in line 238? since we're converting normal/expected cases to IsAWSErr usage anyway?

Reply:
agreed. I missed that one I think.